packages/core/src/browser/addEventListener.ts

```diff
@@ -2,7 +2,11 @@ import { monitor } from '../tools/monitor'
 import { getZoneJsOriginalValue } from '../tools/getZoneJsOriginalValue'
 import type { VisualViewport, VisualViewportEventMap } from './types'
 
-export const enum DOM_EVENT {
+export type TrustableEvent<E extends Event = Event> = E & { __ddIsTrusted?: boolean }
+
+export // We want to use a real enum (i.e. not a const enum) here, to be able to iterate over it to automatically add _ddIsTrusted in e2e tests
+// eslint-disable-next-line no-restricted-syntax
+enum DOM_EVENT {
```

Comment:
comments between `export` and `enum` seem a bit odd, is it intentional?

Reply:
I was IDE auto export :)